rust/core-lib/src/lib.rs

```diff
@@ -124,6 +125,7 @@ use internal::line_cache_shadow;
 use internal::width_cache;
 #[cfg(feature = "ledger")]
 use internal::fuchsia;
+use internal::backspace;
```

Comment:
this 'internal' stuff has been deleted, so you don't need to make this change; maybe rebase to `master`?

Reply:
OK. I have rebased!